atlassian/rest_client.py

```diff
@@ -445,15 +680,15 @@ def put(
 
     def patch(
         self,
-        path,
-        data=None,
-        headers=None,
-        files=None,
-        trailing=None,
-        params=None,
-        absolute=False,
-        advanced_mode=False,
-    ):
+        path: str,
```

Comment:
Not adding the override variations here since `patch` doesn't get used in `jira.py`.